docs/guides/hooks-list.rst

```diff
@@ -454,6 +454,20 @@ Note that not all hooks apply to instant notifications.
 	 * ``headers`` - an array of headers
 	 * ``params`` - other parameters inherited from the notification object or passed directly to ``elgg_send_email()``
 
+**email:message, system**
+	Triggered by ``elgg_send_email()``.
+	Applies to **subscriptions** and **instant** notifications with ``email`` method.
+	This hook is triggered just before the email is being send. This hook allows last minute changes to the contents or structure of the email message - the handler must return a ``\Zend\Mail\Message``.
```

Comment:
This hook allows you to alter an instance of `Zend\Mail\Message` before it is passed to the email transport.